src/index.html

```diff
@@ -5,8 +5,397 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="logo" class="header__logo">
+        <img src="./images/logo.png" alt="logo">
+      </a>
+      <nav class="header__nav">
+        <ul class="header__nav">
+          <li class="header__item">
+            <a href="Apple" class="header__link is-active">Apple</a>
+          </li>
+          <li class="header__item">
+            <a href="Samsung" class="header__link">Samsung</a>
+          </li>
+          <li class="header__item">
+            <a href="Смартфоны и телефоны" class="header__link">Смартфоны и телефоны</a>
+          </li>
+          <li class="header__item">
+            <a
+              href="Ноутбуки и компьютеры"
+              class="header__link"
+              data-qa="nav-hover"
+            >
+              Ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class="header__item">
+            <a href="Гаджеты" class="header__link">Гаджеты</a>
+          </li>
+          <li class="header__item">
+            <a href="Планшеты" class="header__link">Планшеты</a>
+          </li>
+          <li class="header__item">
+            <a href="Фото" class="header__link">Фото</a>
+          </li>
+          <li class="header__item">
+            <a href="Видео" class="header__link">Видео</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="conteiner">
+      <div class="container-catalog">
+        <div
+    class="card"
+    data-qa = "card"
+  >
+          <img
+        class="card__img"
+        src="./images/imac.jpeg"
+        alt="image for mackbook"
+      >
+          <div class="card__desktop">
+            <div class="card__title">
+      Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+            </div>
+            <div class="card_text">
+      Код товара: 195434
+            </div>
+          </div>
+          <div class="card__element">
+            <div class="stars">
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+            </div>
+            <div class="card_review">
```

Comment:
```suggestion
            <div class="card__review">
```
make it a BEM-element, not a block